pkg/resource/model/custom_delta.go

```diff
@@ -16,6 +16,7 @@ package model
 import (
 	ackcompare "github.com/aws-controllers-k8s/runtime/pkg/compare"
 	svcapitypes "github.com/aws-controllers-k8s/sagemaker-controller/apis/v1alpha1"
+	"github.com/aws/aws-sdk-go/aws"
```

Comment:
Please apply `goimport` here

Reply:
I downloaded and ran goimport but didn't seem to change anything? What is the expected output ?

Reply:
last I tried, it brought in a bunch of dependencies to the package.

Reply:
same, but is there a package size concern or should we just go ahead and include it ?